package.json

```diff
@@ -44,6 +44,7 @@
     "@sentry/node": "7.41.0",
     "@sentry/tracing": "7.41.0",
     "@sentry/types": "7.41.0",
+    "@shopify/address": "4.0.6",
```

Comment:
1.8kB minified (is using a request to Shopify API to get formatting data), same library as used in frontend. Used to create a formatted address string that is correct for the country from structured data.